.asf.yaml

```diff
@@ -15,12 +35,19 @@ github:
     wiki: false
     issues: true
     projects: false
+
   collaborators:  # Note: the number of collaborators is limited to 10
     - vinooganesh
+    - alamb
 
 staging:
   profile: ~
   whoami:  asf-staging
  
 publish:
-  whoami:  asf-site
\ No newline at end of file
+  whoami:  asf-site
+
+notifications:
+  commits:      commits@parquet.apache.org
+  issues:       issues@parquet.apache.org
+  pullrequests: issues@parquet.apache.org
```

Comment:
Should this go to issues? I think if we don't want it going to dev we can have it go to commits@ also. This does seem like it's the default though, so no strong feelings

Reply:
I don't have a preference either. Let me put it into commits@ for now.